addons/web/static/src/core/user_switch/user_switch.js

```diff
@@ -0,0 +1,64 @@
+import { Component, useState } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { lastUsers, registerUsers } from "@web/core/user";
+import { imageUrl } from "@web/core/utils/urls";
+
+export class UserSwitch extends Component {
+    static template = "web.login_user_switch";
+    static props = {};
+
+    setup() {
+        const users = lastUsers();
+        this.state = useState({
+            users,
+            displayUserChoice: users.length,
+        });
+        this.form = document.querySelector("form.oe_login_form");
+        if (users.length) {
+            this._maskInputForm();
+        } else {
+            this._unmaskInputForm();
+            this.form.querySelector("input#login").focus();
+        }
```

Comment:
Same logic at line 35 to 40 maybe refactor this into a function with a bool as parameter